index.js

```diff
@@ -950,6 +957,7 @@ module.exports = class Hypercore extends EventEmitter {
   }
 
   download (range) {
+    this.tracer.trace()
```

Comment:
move this one to _download and trace after opening and pass the range, { range } to props